docs/src/modules/components/AppTableOfContents.js

```diff
@@ -64,6 +34,7 @@ const styles = theme => ({
   },
   contents: {
     marginTop: theme.spacing(2),
+    paddingLeft: theme.spacing(1.5),
```

Comment:
Align the heading with the TOC, taking into account the left border and padding.